extensions/amp-ad/0.1/amp-ad.js

```diff
@@ -76,8 +76,8 @@ export class AmpAd extends AMP.BaseElement {
 
       // TODO(tdrl): Check amp-ad registry to see if they have this already.
       if (!a4aRegistry[type] ||
-          !a4aRegistry[type](this.win, this.element) ||
-          this.win.document.querySelector('meta[name=amp-3p-iframe-src]')) {
+          this.win.document.querySelector('meta[name=amp-3p-iframe-src]') ||
```

Comment:
Nit: I would put this check very first, so that the `a4aRegistry[type]` goes right beside `a4aRegistry[type](this.win, this.element)`, which is the more common idiom I think.